quick-search-engine/README.md

```diff
@@ -0,0 +1,116 @@
+# Qdrant Search Engine
```

Comment:
Add a pyproject.toml using Poetry or similar file which has all the dependencies perhaps? 

This looks great at first glance and having all the dependencies in one place would make it easier to use it!